components/steps/nz-steps.spec.ts

```diff
@@ -322,3 +476,31 @@ export class NzTestStepForComponent {
     this.steps.push(4);
   }
 }
+
+@Component({
+  selector: 'nz-test-onpush-outer-steps',
+  template: `
+    <nz-steps [nzCurrent]="current" [nzDirection]="direction" [nzLabelPlacement]="labelPlacement" [nzSize]="size" [nzStatus]="status" [nzProgressDot]="progressDot" [nzStartIndex]="startIndex">
+      <nz-step nzTitle="0title" nzDescription="0description"></nz-step>
+      <nz-step nzTitle="1title" nzDescription="1description"></nz-step>
+      <nz-step nzTitle="2title" nzDescription="2description"></nz-step>
+    </nz-steps>
+    <ng-template #progressTemplate let-dot let-status="status" let-index="index">
+      <span class="insert-span">{{status}}{{index}}</span>
+      <ng-template [ngTemplateOutlet]="dot"></ng-template>
+    </ng-template>
+  `,
+  changeDetection : ChangeDetectionStrategy.OnPush
+})
+export class NzTestOnPushOuterStepsComponent {
```

Comment:
You can just make old test components onpush. It seems redundant to testing both default and onpush.